src/main/steps/applicant2/confirm-your-joint-application/post.ts

```diff
@@ -0,0 +1,19 @@
+import autobind from 'autobind-decorator';
+import { NextFunction, Response } from 'express';
+
+import { APPLICANT_2_APPROVE } from '../../../app/case/definition';
+import { AppRequest } from '../../../app/controller/AppRequest';
+import { AnyObject, PostController } from '../../../app/controller/PostController';
+import { Form } from '../../../app/form/Form';
+
+@autobind
+export default class ConfirmYourJointApplicationPostController extends PostController<AnyObject> {
+  constructor(protected readonly form: Form) {
+    super(form);
+  }
+
+  public async post(req: AppRequest<AnyObject>, res: Response, next?: NextFunction): Promise<void> {
+    const event = APPLICANT_2_APPROVE;
+    await super.post(req, res, next, event);
```

Comment:
probably no need to create the const `event`. just pass the APPLICANT_2_APPROVE directly to the method?

Reply:
```suggestion
    await super.post(req, res, next, APPLICANT_2_APPROVE);
```